v3/pixelspace/shadow80/shadow80.json

```diff
@@ -0,0 +1,319 @@
+        },
+        "2,13\n\n\n0,0",
+       
+        {
+          "x": 0.5,
+          "c": "#aaaaaa"
+        },
+        "2,14",
+        "2,15",
+        "2,16",
+        {
+          "x": 1.75,
+          "w":1.25,
+          "h":2,
+          "c": "#777777",
+          "w2":1.5,
+          "h2":1,
+          "x2":-0.25
+        },
+        "3,13\n\n\n0,1"
+      ],
+      [
+        {
+          "x": 2.5,
+          "c": "#aaaaaa",
+          "w": 1.75
+        },
+        "3,0",
+        {
+            "c": "#cccccc"
+        },
+        "3,1",
+        "3,2",
+        "3,3",
+        "3,4",
+        "3,5",
+        "3,6",
+        "3,7",
+        "3,8",
+        "3,9",
+        "3,10",
+        "3,11",
+        {
+          "w": 2.25, 
+          "c": "#777777"
+        },
+        "3,13\n\n\n0,0",
+        {
+          "x": 4.25,
+          "c": "#cccccc"
+        },
+        "2,13\n\n\n0,1"
+        
+       
+      ],
+      [
+        {
+          "c": "#aaaaaa",
+          "w": 1.25
+        },
+        "4,0\n\n\n2,1",
+        {
+          "c": "#cccccc"
+        },
+        "4,1\n\n\n2,1",
+        {
+          "x": 0.25,
+          "c": "#aaaaaa",
+          "w": 2.25
+        },
+        "4,0\n\n\n2,0",
+        {
+          "c": "#cccccc"
+        },
+        "4,2",
+        "4,3",
+        "4,4",
+        "4,5",
+        "4,6",
+        "4,7",
+        "4,8",
+        "4,9",
+        "4,10",
+        "4,11",
+        {
+          "c": "#aaaaaa",
+          "w":2.75
+        },
+        "4,12\n\n\n3,0",
+        {
+          "x": 1.25
+        },
+        "4,15",
+        {
+          "x": 1.25,
+          "w":1.75
+        },
+        "4,12\n\n\n3,1",
+        "4,13\n\n\n3,1"
+      ],
+      [
+        {
+          "x": 2.5,
+          "w": 1.5
+        },
+        "5,0\n\n\n1,0",
+        "5,1\n\n\n1,0",
+        {
+          "w":1.5
+        },
+        "5,2\n\n\n1,0",
+        {
+          "c": "#cccccc",
+          "w": 7
+        },
+        "5,6\n\n\n1,0",
+        {
+          "c": "#aaaaaa",
+          "w":1.5
+        },
+        "5,11\n\n\n1,0",
+        "5,12\n\n\n1,0",
+        {
+          "w":1.5
+        }, 
+       "5,13\n\n\n1,0",
+        {
+          "x": 0.25
+        },
+        "5,14",
+        "5,15",
+        "5,16"
+      ],
+      [
+        {
+          "y": 0.25,
+          "x": 2.5,
+          "w": 1.25
+        },
+        "5,0\n\n\n1,1",
+        {
+          "w":1.25
+        },
+        "5,1\n\n\n1,1",
+        {
+          "w":1.25
+        },
+        "5,2\n\n\n1,1",
+        {
+          "c": "#cccccc",
+          "w": 6.25
+        },
+        "5,6\n\n\n1,1",
+        {
+          "c": "#aaaaaa",
+          "w":1.25
+        },
+        "5,10\n\n\n1,1",
+        {
+          "w":1.25
+        },
+        "5,11\n\n\n1,1",
+        {
+          "w":1.25
+        },
+        "5,12\n\n\n1,1",
+        {
+          "w":1.25
+        },
+        "5,13\n\n\n1,1"
+      ]
+    ]
+
+
+
+
+  }
+}
```

Comment:
Formatting...... yet again.....
```suggestion
{
  "name": "SHADOW80",
  "vendorId": "0x0011",
  "productId": "0x0011",
  "matrix": {
    "rows": 6,
    "cols": 17
  },
  "customKeycodes": [
    {
      "name": "Toggle WinKey Lock",
      "title": "Toggle WinKey Lock",
      "shortName": "WinLock"
    },
    {
      "name": "Toggle macOS Keymap",
      "title": "Toggle macOS Keymap",
      "shortName": "TogMacOS"
    },
    {
      "name": "Mission Control",
      "title": "Mission Control in macOS",
      "shortName": "Mission Control"
    },
    {
      "name": "Launch Pad",
      "title": "Launch Pad in macOS",
      "shortName": "Launch Pad"
    }
  ],
  "keycodes": [
    "qmk_lighting"
  ],
  "menus": [
    "qmk_rgb_matrix"
  ],
  "layouts": {
    "labels": [
      "ISO Enter",
      [
        "Bottom Row",
        "7U",
        "6.25U"
      ],
      "Split Lshift",
      "Split Rshift",
      "Split BackSpace"
    ],
    "keymap": [
      [
        {
          "x": 2.5,
          "c": "#777777"
        },
        "0,0",
        {
          "x": 0.25,
          "c": "#cccccc"
        },
        "0,1",
        "0,2",
        "0,3",
        "0,4",
        {
          "x": 0.25,
          "c": "#aaaaaa"
        },
        "0,5",
        "0,6",
        "0,7",
        "0,8",
        {
          "x": 0.25,
          "c": "#cccccc"
        },
        "0,9",
        "0,10",
        "0,11",
        "0,12",
        {
          "x": 0.25,
          "c": "#aaaaaa"
        },
        "0,13",
        {
          "x": 0.25
        },
        "0,14",
        "0,15",
        "0,16"
      ],
      [
        {
          "y": 0.25,
          "x": 2.5,
          "c": "#cccccc"
        },
        "1,0",
        "1,1",
        "1,2",
        "1,3",
        "1,4",
        "1,5",
        "1,6",
        "1,7",
        "1,8",
        "1,9",
        "1,10",
        "1,11",
        "1,12",
        {
          "c": "#aaaaaa",
          "w": 2
        },
        "1,13\n\n\n4,0",
        {
          "x": 0.5
        },
        "1,14",
        "1,15",
        "1,16",
        {
          "x": 1,
          "c": "#cccccc"
        },
        "3,12\n\n\n4,1",
        "1,13\n\n\n4,1"
      ],
      [
        {
          "x": 2.5,
          "c": "#aaaaaa",
          "w": 1.5
        },
        "2,0",
        {
          "c": "#cccccc"
        },
        "2,1",
        "2,2",
        "2,3",
        "2,4",
        "2,5",
        "2,6",
        "2,7",
        "2,8",
        "2,9",
        "2,10",
        "2,11",
        "2,12",
        {
          "w": 1.5
        },
        "2,13\n\n\n0,0",
        {
          "x": 0.5,
          "c": "#aaaaaa"
        },
        "2,14",
        "2,15",
        "2,16",
        {
          "x": 1.75,
          "c": "#777777",
          "w": 1.25,
          "h": 2,
          "w2": 1.5,
          "h2": 1,
          "x2": -0.25
        },
        "3,13\n\n\n0,1"
      ],
      [
        {
          "x": 2.5,
          "c": "#aaaaaa",
          "w": 1.75
        },
        "3,0",
        {
          "c": "#cccccc"
        },
        "3,1",
        "3,2",
        "3,3",
        "3,4",
        "3,5",
        "3,6",
        "3,7",
        "3,8",
        "3,9",
        "3,10",
        "3,11",
        {
          "c": "#777777",
          "w": 2.25
        },
        "3,13\n\n\n0,0",
        {
          "x": 4.25,
          "c": "#cccccc"
        },
        "2,13\n\n\n0,1"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "4,0\n\n\n2,1",
        {
          "c": "#cccccc"
        },
        "4,1\n\n\n2,1",
        {
          "x": 0.25,
          "c": "#aaaaaa",
          "w": 2.25
        },
        "4,0\n\n\n2,0",
        {
          "c": "#cccccc"
        },
        "4,2",
        "4,3",
        "4,4",
        "4,5",
        "4,6",
        "4,7",
        "4,8",
        "4,9",
        "4,10",
        "4,11",
        {
          "c": "#aaaaaa",
          "w": 2.75
        },
        "4,12\n\n\n3,0",
        {
          "x": 1.25
        },
        "4,15",
        {
          "x": 1.5,
          "w": 1.75
        },
        "4,12\n\n\n3,1",
        "4,13\n\n\n3,1"
      ],
      [
        {
          "x": 2.5,
          "w": 1.5
        },
        "5,0\n\n\n1,0",
        "5,1\n\n\n1,0",
        {
          "w": 1.5
        },
        "5,2\n\n\n1,0",
        {
          "c": "#cccccc",
          "w": 7
        },
        "5,6\n\n\n1,0",
        {
          "c": "#aaaaaa",
          "w": 1.5
        },
        "5,11\n\n\n1,0",
        "5,12\n\n\n1,0",
        {
          "w": 1.5
        },
        "5,13\n\n\n1,0",
        {
          "x": 0.25
        },
        "5,14",
        "5,15",
        "5,16"
      ],
      [
        {
          "y": 0.25,
          "x": 2.5,
          "w": 1.25
        },
        "5,0\n\n\n1,1",
        {
          "w": 1.25
        },
        "5,1\n\n\n1,1",
        {
          "w": 1.25
        },
        "5,2\n\n\n1,1",
        {
          "c": "#cccccc",
          "w": 6.25
        },
        "5,6\n\n\n1,1",
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "5,10\n\n\n1,1",
        {
          "w": 1.25
        },
        "5,11\n\n\n1,1",
        {
          "w": 1.25
        },
        "5,12\n\n\n1,1",
        {
          "w": 1.25
        },
        "5,13\n\n\n1,1"
      ]
    ]
  }
}

```

Reply:
<img width="226" alt="Screenshot 2024-04-28 at 12 28 41 PM" src="https://github.com/the-via/keyboards/assets/40441626/8f3b3717-87df-4b47-99e9-ba04b1972b61">
🧐